contracts/RoyaltyFeeManager.sol

```diff
@@ -43,16 +74,70 @@ contract RoyaltyFeeManager is
         uint256 tokenId,
         uint256 amount
     ) external view override returns (address, uint256) {
+        return _calculateRoyaltyFeeAndGetRecipient(collection, tokenId, amount);
+    }
+
+    /**
+     * @notice Calculate royalty fee amount parts
+     * @param _collection address of the NFT contract
+     * @param _tokenId tokenId
+     * @param _amount amount to transfer
+     */
+    function calculateRoyaltyFeeAmountParts(
+        address _collection,
+        uint256 _tokenId,
+        uint256 _amount
+    ) external view override returns (RoyaltyFeeTypes.FeeAmountPart[] memory) {
+        if (address(royaltyFeeRegistryV2) == address(0)) {
+            // TODO: Should this revert or return empty array?
+            revert RoyaltyFeeManager__RoyaltyFeeRegistryV2NotSet();
```

Comment:
anyone have thoughts on whether this should revert or just return empty array? cc @LouisMeMyself